Capstone.pro

```diff
@@ -1,7 +1,6 @@
-QT += \
-    core gui \
-    multimedia
-
+QT       += core gui
+QT += multimedia
+QT += core gui printsupport multimedia
```

Comment:
could you revert back to the format in master? maybe you need to merge from master first

Reply:
changed to fit the format in master